monkey/monkey_island/cc/ui/src/services/AuthService.js

```diff
@@ -109,12 +147,32 @@ export default class AuthService {
           res.clone().json().then(res_json => {
             console.log('Got 401 from server while trying to authFetch: ' + JSON.stringify(res_json));
           });
-          this._removeToken();
+          this._removeToken(this.AUTH_TOKEN_NAME_IN_LOCALSTORAGE);
```

Comment:
The token is being removed here. What happens when line 68 is executed then?

Reply:
Nothing much, two different things. Ff for some reason we have the auth token still in the storage then we need to remove it on 401 from the `refresh-authentication-token` endpoint. The one on 150 is if we have 401 on whatever other request.

Reply:
> for some reason we have the auth token still in the storage then we need to remove it on 401 from the refresh-authentication-token endpoint.

What happens when it's already removed (because of line 150)? Does it crash?

Reply:
If the token doesn't exist it won't do anything

Reply:
Nope, the localstorage can't find it so nothing to delete.